inputfiles/browser.webidl.preprocessed.json

```diff
@@ -8291,6 +8291,13 @@
                             "default": "\"\"",
                             "type": "DOMString",
                             "type-original": "DOMString"
+                        },
+                        "disposition": {
+                            "specs": "csp",
+                            "name": "disposition",
+                            "default": "\"\"",
+                            "type": "DOMString",
+                            "type-original": "DOMString"
                         }
                     }
```

Comment:
Was this manually added to the JSON? It might get removed by the next update

Reply:
Hello @orta, yes I manually added otherwise `npm run test` fails. I am totally new to this, if you can tell me what is the correct way  to do it I will be grateful.

Reply:
This is the error I get:
<img width="1929" alt="image" src="https://user-images.githubusercontent.com/11148626/123790346-80318380-d8de-11eb-98e9-76c605ea59a6.png">

Reply:
There's some good docs in the README on how to update an IDL: https://github.com/microsoft/TypeScript-DOM-lib-generator#contribution-guidelines

Reply:
Fixed it, thanks.